ppoprf/src/ppoprf.rs

```diff
@@ -68,6 +70,66 @@ impl ProofDLEQ {
     Self { c: chl, s }
   }
 
+  fn new_batch(
```

Comment:
@rillian I've added [tests](https://github.com/brave/sta-rs/blob/batch-dleq-proofs/ppoprf/src/ppoprf.rs#L762) now but still get the unused function warnings (for `new_batch`, `verify_batch` and `compute_composites`). I assume the tests alone don't suppress the warnings.

Reply:
Ah, right. It's still warning because they're not called from any exported API function. I suggest updating the `Client` and `Server` objects to call the `_batch` functions instead of the single-point versions. It's probably ok to have those interfaces always take `&[Point]` instead of `&Point` and let callers with only one point array it up first.

Sorry for the bad advice.